### PR TITLE
JBPM-5996: Click is not ignored in actions column on list views

### PR DESCRIPTION
**Pull request by rbarriuso**

### CODE REVIEW

**Comment by ederign** [comment]
ok to test

**Comment by cristianonicolai** [comment]
jenkins please retest this.

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java [hunk 1]
```diff
@@ -399,5 +400,20 @@ public int getRefreshValue() {
     public void saveRefreshValue( int newValue ) {
         filterPagedTable.saveNewRefreshInterval( newValue );
     }
+    
+    public void addSelectionIgnoreColumn(Column<T, ?> column){
+        ignoreSelectionColumns.add(column);
+    }
+    
+    public boolean isSelectionIgnoreColumn(int colIx){
+        boolean ret = false;
```
Comment (cristianonicolai) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java]: I would suggest using a stream like alternative: .stream.filter.findFirst.isPresent or similar.

Comment (rbarriuso) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java]: Good idea

**Comment by rbarriuso** [comment]
I've rebased, added tests and moved the "SelectionIgnoreColumn" functionality to the `ExtendedPagedTable` class, as I realized that it fits better there than in `AbstractMultiGridView`.

**Comment by rbarriuso** [comment]
Applied functionality to `ProcessDefinitionListViewImpl`

**Pull request merged**